code-challenges.js

```diff
@@ -9,41 +9,131 @@
+// const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 
-const stockExchange = [13, 34, -4, 42, 5, -5, 10, 27, 42, 10]
-// Expected output: [42, 5, -5]
+// function sumRuns(numbers){
+// //create a function called sumRuns that takes numbers
+//     let freshArr = 0;
+// //set freshArr value to 0 to add numbers later    
+//     for (let i=0; i < numbers.length; i++){
+// //iterate through each number in numbers        
+//         freshArr += numbers[i];
+// //add each elements to freshArr        
+//     }
+//     return freshArr
+// // return sum     
+// } 
 
-const cohortList = ["Alpha", "Bravo", "Charlie", "Delta", "Echo", "Foxtrot"]
-// Expected output: ["Delta", "Echo", "Foxtrot"]
+// console.log(sumRuns(padres1998WorldSeriesRuns))
+   
 
-// Explain your code:
+// // Expected output: 17
+
+
+// // Explain your code:
+// I explained as i did it
+
+// // --------------------2) Create a function that takes in a string and  returns the string with all the characters in reversed order. Use the test variable provided below.
+
+// const lesMiserables = "Jean Valjean"
+
+// function reverseMiserables(str) {
+// //create function called reverseMiserables that takes string        
+//     return str.split('').reverse().join('')
+// // split string into an array, reverse the array then convert array back to a string    
+// }
+// console.log(reverseMiserables(lesMiserables))
+
+// // Expected output: "naejlaV naeJ"
+
+// const lionKing = "Simba"
+
+// function reverseSimba(strAgain) {
+//     return strAgain.split('').reverse().join('')
+// }
+// console.log(reverseSimba(lionKing))
+
+
+// // Expected output: "abmiS"
+
+// // Explain your code: 
+//create a funct that can take in a string
+//split string into an array, reverse the array then convert array back to a string  
+
+// // --------------------3) Create a function that takes in an array and returns an array with only the items from the third, fourth, and fifth indexes. Use the test variable provided below.
+
+// const stockExchange = [13, 34, -4, 42, 5, -5, 10, 27, 42, 10]
+
+// function thrFourFifth(arr) {
+// //create a funct that takes in array as argument    
+//    return arr.slice(2, 5)
+// // use slice on array to grab every index between 2 and 5 then returns a new array    
+// }
+// console.log(thrFourFifth(stockExchange))
+// //call/invoke the funct
+
+// // // Expected output: [42, 5, -5]
+
+// const cohortList = ["Alpha", "Bravo", "Charlie", "Delta", "Echo", "Foxtrot"]
+
+// function delEchFox(str) {
+// //create a funct called delEchFox that takes strings    
+//     return str.slice(3, 6)
+// //use split to return indexes 3, 6 and every index between    
+// }
+// console.log(delEchFox(cohortList))
+// //now call/invoke 
+
+// // Expected output: ["Delta", "Echo", "Foxtrot"]
+
+// // Explain your code:
+//give variables parameters capable of holding strings
+//use the .slice method to pull specific strings in the index out from between two points in the array
```

Comment:
Good work, Amir!  Just a couple of things - you don't have to write a separate function for each test variable in the challenge.  The idea is that all the variables will work in your function - so you did a bit of extra work. 😁
Also be sure to run your code - in the last challenge, your first function does not give the desired output.  Your code and logic look good though!